cody/animate.js

```diff
@@ -0,0 +1,11 @@
+const userAction = async () => { 
+  const response = await fetch("http://localhost:5000/patients?count=20");
+  const myJson = await response.json(); 
+
+  myJson.forEach(element => {
+    console.log(element.firstName);
+    var root = document.getElementById("root");
+    var html = root.innerHTML;
+    root.innerHTML += '</div><div class="display"><div> ' + element.firstName + '</div><div> ' + element.email + ' </div><div> ' + element.nhsNumber + ' </div></div>';
```

Comment:
If you are looking at making an improvement to the user interface. You could add a section to the top of the page for patient cards. see https://www.w3schools.com/howto/howto_css_contact_chips.asp